libs/api/domains/auth-admin/src/lib/client/clients.service.ts

```diff
@@ -143,21 +142,13 @@ export class ClientsService extends MultiEnvironmentService {
       }),
     )
 
-    return created
-      .map((resp, index) => {
-        if (resp.status === 'fulfilled' && resp.value) {
-          return {
-            clientId: resp.value.clientId,
-            environment: input.environments[index],
-          }
-        } else if (resp.status === 'rejected') {
-          this.logger.error(
-            `Failed to create application ${input.clientId} in environment ${input.environments[index]}`,
-            resp.reason,
-          )
-        }
-      })
-      .filter(isDefined)
+    return this.handleSettledPromises(settledPromises, {
```

Comment:
Nice! We should make a nice to have task to use this handler as well in the `getClients` and `getClientById`

Reply:
https://www.notion.so/Refactor-code-in-client-service-to-replicate-code-in-scope-service-e655090a677d44c59073fe6b55177251?pvs=4